docs/_posts/2022-11-30-luau-recap-november-2022.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: single
+title:  "Luau Recap: November 2022"
+---
+
+While the team is busy to bring some bigger things in the future, we have made some small improvements this month.
+
+[Cross-posted to the [Roblox Developer Forum](https://devforum.roblox.com/t/luau-recap-november-2022/).]
+
+## Analysis improvements
+
+We have improved tagged union type refinements to only include unhandled type cases in the `else` branch of the `if` statement:
+
+```lua
+type Ok<T> = { tag: "ok", value: T }
+type Err = { tag: "error", msg: string }
+type Result<T> = Ok<T> | Err
+
+function unwrap<T>(r: Result<T>): T?
+    if r.tag == "ok" then
+        return r.value
+    else
+        -- Luau now understands that 'r' here can only be the 'Err' part
+        print(r.msg)
+        return nil
+    end
+end
+```
+
+For better inference, we updated definition of `Enum.SomeType:GetEnumItems()` to return `{Enum.SomeType}` instead of common `{EnumItem}` and the return type of `next` function now includes the possibility of key being `nil`.
```

Comment:
```suggestion
For better inference, we updated the definition of `Enum.SomeType:GetEnumItems()` to return `{Enum.SomeType}` instead of common `{EnumItem}` and the return type of `next` function now includes the possibility of key being `nil`.
```